apple/internal/templates/apple_simulator.template.py

```diff
@@ -129,6 +131,9 @@ def is_apple_tv(self) -> bool:
 
   def is_apple_watch(self) -> bool:
     return self.has_product_family_or_identifier("Apple Watch")
+  
+  def is_apple_vision(self) -> bool:
+    return self.has_product_family_or_identifier("Apple Vision")
```

Comment:
I went with "Apple Vision" instead of "Apple Vision Pro" to make this more future-proof for a possible non-Pro device 🙃